DebugServer/stdafx.h

```diff
@@ -19,3 +19,16 @@
 #include <Shlobj.h>
 
 #define BOOST_ALL_NO_LIB = 1
+
+// Ruby was configured for /Md builds. These macros are needed to
+// prevent compiler errors with Visual Studio when builds are compiled with MT.
+
+// Ruby 2.0
```

Comment:
Originally I had it before the <ruby.h> include headers. But that didn't work. Adding it to `stdafx.h` ensured it work. I started wondering if `ruby.h` got included somewhere else as well, but I didn't find it. I meant to ask around if anyone had any ideas to why.